### PR TITLE
Smaato: Split multiple media types

### PR DESCRIPTION
**Pull request by el-chuck**
Split multiple media types into individual requests

### CODE REVIEW

**Review by AlexBVolcy — APPROVED** [review]
Looks good to me!

File: adapters/smaato/smaato.go [hunk 1]
```diff
@@ -436,7 +465,7 @@ func setImpForAdspace(imp *openrtb2.Imp) error {
 		return nil
 	}
 
-	return &errortypes.BadInput{Message: "Invalid MediaType. Smaato only supports Banner and Video."}
+	return nil
```
Comment (guscarreon) [inline adapters/smaato/smaato.go:468]: Newly created `splitImpressionsByMediaType(...)` function and the `return nil` statements in lines 459 and 465 make this line unreachable. It doesn't affect anything and I don't have a problem with it so, if you like to fix it in this PR, ok, but I will approve your PR either way. Your code looks pretty solid.

**Review by guscarreon — APPROVED** [review]
I left a comment in regards a small nitpick but it's so small that it's up to you @el-chuck if you think it's worth addressing or not. These modifications look good to me.

Approved

File: adapters/smaato/smaato.go (hunk 1 above)
Comment (el-chuck) [reply adapters/smaato/smaato.go:468]: ok, we will address this with next PR

Comment (guscarreon) [reply adapters/smaato/smaato.go:468]: Awesome @el-chuck . Thank you for addressing our feedback

**Pull request merged**